src/frontend/src/lib/i18n/en.json

```diff
@@ -89,7 +89,7 @@
 			"all_dapps": "All dapps",
 			"featured": "Featured",
 			"sign_in": "Sign-in to explore the ecosystem.",
-			"title": "Explore Web3 Ecosystem",
+			"title": "Explore apps",
```

Comment:
not uppercase since it is a title?

`Explore Apps`

Reply:
@StefanBerger-DFINITY 
ticket for reference: https://dfinity.atlassian.net/browse/OISY-276

Reply:
Agree on the capitalization, if the UI guideline requires this for titles. So "Explore Apps"
About the Apps/Dapps: I got this exact text requested.

Reply:
adjusted